### ISSUE TITLE
Fix font issues with new poster templates

### ISSUE THREAD

**Issue by jflory7** [RITlug/tasks#37]
There are some weird kinks with Overpass font loading and the new poster template. It makes it difficult to load the right font across different operating systems / devices.

@ct-martin has a lead on the issue, but needs to look into it more, mostly on horizontal template.

This is a low priority task since it does not block new posters, but it would be nice to have corrected.

**Comment by Tjzabel** [comment]
@jflory7 has this issue been fixed?

**Comment by justwheel** [comment]
@Tjzabel This is something @ct-martin is working on when he has time. It's a low priority task, so it may happen next semester. I'm not sure if we're planning on any other posters this semester anyways.

**Comment by Tjzabel** [comment]
@ct-martin Is this something we need to worry about anymore?

**Comment by ct-martin** [comment]
@Tjzabel yes. I've also run into Inkscape doing weird things unrelated to this issue but also preventing me from working on it so I've pinged @gen1e for possible help if she has bandwidth (she will recalculate her bandwidth this weekend). I can create posters in non-FOSS software in the mean time, but for now anyone else will run into issues.
